data/participants.json

```diff
@@ -448,5 +448,15 @@
             "twitter"  : "",
             "referral" : ""
 	    }
```

Comment:
The json file is not valid. Check the syntax and make the required changes. Use [this](https://jsonlint.com/)